kytos/cli/commands/napps/parser.py

```diff
@@ -75,11 +76,15 @@ def parse_napps(napp_args):
 
     def parse_napp(arg):
         """Parse one argument."""
```

Comment:
please, add version to return value in outer function and explain it can be None if it was not specified by the user.